daq/host.py

```diff
@@ -61,6 +61,7 @@ class ConnectedHost:
     """Class managing a device-under-test"""
 
     _MONITOR_SCAN_SEC = 30
+    _DEFAULT_TIMEOUT = 350
```

Comment:
we'll need a way to apply a global setting to this without changing code.  See _MONITOR_SCAN_SEC and how it pulls a value from config, and then add a similar commented-out example to misc/system.conf (search for monitor_scan_sec). Otherwise, we run into the situation where we might break existing installs, which is OK if there's an easy way to recover (e.g. adding a global config).